samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -545,7 +545,9 @@ static void stream_recv(struct bt_bap_stream *stream,
 			const struct bt_iso_recv_info *info,
 			struct net_buf *buf)
 {
-	printk("Incoming audio on stream %p len %u\n", stream, buf->len);
+	if (info->flags & BT_ISO_FLAGS_VALID) {
+		printk("Incoming audio on stream %p len %u\n", stream, buf->len);
+	}
```

Comment:
I considered that. Then we would have to add som logic to only print a every N reports otherwise the logs would be flooded with such messages as such reports are raised every SDU interval the peer is not providing data